stable/datadog/templates/daemonset.yaml

```diff
@@ -12,6 +12,7 @@ spec:
         app: {{ template "fullname" . }}
       name: {{ template "fullname" . }}
     spec:
+      hostNetwork: true
```

Comment:
To be confirmed by someone more seasoned than me, but I think hostPort (having a port redirection to the container) is enough for this use case and hostNetwork (sharing the host's IP) is not necessary.